### PR TITLE
fix: add newer version of cachet

### PR DESCRIPTION
**Pull request by linuxbandit**

### CODE REVIEW

File: oms-status/docker/cachet/.env.example [hunk 1]
```diff
@@ -43,3 +43,8 @@ NEXMO_SMS_FROM=Cachet
 
 TRUSTED_PROXIES=
 
+SLACK_WEBHOOK_URL=
```
Comment (serge1peshcoff) [inline oms-status/docker/cachet/.env.example]: do we have the provisioning for this file?

Comment (linuxbandit) [reply oms-status/docker/cachet/.env.example]: we could, good idea

**Comment by linuxbandit** [comment]
https://github.com/CachetHQ/Cachet/commit/6e9d3563ef7f76e40e28d09aa2a81a8c889586ee

comes from cachet issue 3441... apparently it's now broken because that link is a redirect to twitter and not a rss feed anymore..........

**Comment by linuxbandit** [comment]
or not? apparently it is a cache (i.e. redis) problem, and dashboard can only be seen after `./helper.sh --docker -- exec cachet php artisan cache:clear`

**Review by serge1peshcoff — APPROVED** [review]

**Pull request merged**